### PR TITLE
Translate Slovak.java

### PR DESCRIPTION
**Pull request by BramboraSK**
Slovenčina

### CODE REVIEW

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/LanguageIdentifier.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 	Català(Catalan.class),
 	Slovenščina(Slovenian.class),
 	Schwiizerdütsch(Swissgerman.class),
+	Slovenčina(Slovak.class),
```
Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/LanguageIdentifier.java]: Is "Slovenčina" fine here?

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/Slovak.java [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+		 * "Nichts leichter als das!", which translates literally to
+		 * "Nothing easier than that!", which refers to German being my first
+		 * language.
+		 * 
+		 * Tillerino
+		 * 
+		 * P.S. you can put a link to your profile into the line like this:
+		 * [https://osu.ppy.sh/u/2070907 Tillerino]
+		 */
+		return new Message("[https://osu.ppy.sh/u/BramboraSK BramboraSK] mi pomohol naučiť sa slovensky.");
```
Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/lang/Slovak.java]: The numeric user id will work for longer :)

```suggestion
		return new Message("[https://osu.ppy.sh/u/8543258 BramboraSK] mi pomohol naučiť sa slovensky.");
```

**Comment by Tillerino** [comment]
@omkelderman if merge this, make sure to squash. Otherwise we'll have commits where we deleted and added Czech, which clogs up the history.

File: tillerinobot/src/main/java/tillerino/tillerinobot/lang/LanguageIdentifier.java (hunk 1 above)
Comment (BramboraSK) [reply tillerinobot/src/main/java/tillerino/tillerinobot/lang/LanguageIdentifier.java]: Well, if you want it in alphabetical order, it's Slovenčina, Slovenščina, Schwiizerdutsch, if u don't want it like that, this is okay :D

Comment (Tillerino) [reply tillerinobot/src/main/java/tillerino/tillerinobot/lang/LanguageIdentifier.java]: Okay, thx

**Pull request merged**